README.md

```diff
@@ -25,34 +25,19 @@ The release notes for the `core` module are [here](https://github.com/BitGo/BitG
 
 # NodeJS Version Support Policy
 
-We only support [Long-Term Support](https://github.com/nodejs/Release) versions
-of Node starting with [Node 8.9.0 (LTS)](https://nodejs.org/en/blog/release/v8.9.0).
-
-We specifically limit our support to LTS versions of Node, not because this
-package won't work on other versions, but because we have a limited amount of
-time, and supporting LTS offers the greatest return on that investment. It's
-possible this package will work correctly on newer or older versions of Node.
-
-As each Node LTS version reaches its end-of-life we will remove that version
-from the node engines property of our package's package.json file.  Removing a
-Node version is considered a breaking change and will entail the publishing of
-a new major version of this package. We will not accept any requests to support
-an end-of-life version of Node. Any merge requests or issues supporting an
-end-of-life version of Node will be closed. We will accept code that allows
-this package to run on newer, non-LTS, versions of Node.  Furthermore, we will
-attempt to ensure our own changes work on the latest version of Node. To help
-in that commitment, our continuous integration setup runs the full test suite
-on the latest release of the following versions of node:
-
-* `10`
-
-**Note:** We intend to support the current `lts` (node 12), but it is not currently running due to compability issues with dependencies)
-
-JavaScript package managers should allow you to install this package with any
+Due to constraints from library dependencies, we currenty only support node
+versions `>=10.22.0 <12.0.0`.
+
+**Note:** We intend to support the current `lts` (node 12), in the future.
```

Comment:
I tried `yarn install` node 12 LTS locally: it only failed for `ed25519` which is an optional dependency, so the final state was `success`